test/src/org/labkey/test/tests/response/ResponseSubmissionTest.java

```diff
@@ -110,23 +113,23 @@ public void testMetadata()
         //        4. Invalid Metadata
         //            A. Metadata element missing
         log("Testing Metadata element not present");
-        SubmitResponseCommand cmd = new SubmitResponseCommand(this::log, null, null, appToken, BASE_RESULTS);
+        SubmitResponseCommand cmd = new SubmitResponseCommand(this::log, null, null, null, appToken, BASE_RESULTS);
         cmd.execute(400);
-        assertEquals("Unexpected error message", SubmitResponseCommand.METADATA_MISSING_MESSAGE, cmd.getExceptionMessage());
+        assertEquals("Unexpected error message: " + cmd.getExceptionMessage(), SubmitResponseCommand.METADATA_MISSING_MESSAGE, cmd.getExceptionMessage());
```

Comment:
This message is redundant. `assertEquals` will include the expected and actual values in its error message.
```suggestion
        assertEquals("Unexpected error message.", SubmitResponseCommand.METADATA_MISSING_MESSAGE, cmd.getExceptionMessage());
```

Ditto for the subsequent `assertEquals`s

Reply:
Of course you're right... I hit one of the `assertTrue` cases which needed this change, but I wasn't careful with my search and replace. I've reverted all the `assertEquals` cases.